content/2022-munich/submit.md

```diff
@@ -4,7 +4,7 @@ title: Submit Talk
 
 ## Submit a talk
 
-The PromCon Call for Papers (CfP) will open soon!
+The [PromCon Call for Papers](https://docs.google.com/forms/d/e/1FAIpQLSdkcpMXhT4QAZ9g7IWE74OO08Re0b5R4vjVoXxa2Gidc28hIg/viewform) is open until 2022-09-02, 23:59 UTC
```

Comment:
I would make a big button out of it again like in previous years, that way it's a way bigger call to action and easier to find.

Reply:
Do you want to make the button? Jumping in and out of a social thing, at the moment.

Reply:
Ah sorry, didn't see this anymore yesterday. You can find the 2020 button here: https://raw.githubusercontent.com/promcon/website/main/content/2020-online/register.md